_includes/footer.html

```diff
@@ -1,6 +1,9 @@
 <footer>
+    
     <h5>We love our sponsors!</h5>
-    <p>Please consider <a href="http://1418.team/sponsorship">sponsoring us</a>!</p>
+    <div class="emphasized">
+        <p>Please consider <a href="http://1418.team/sponsorship">sponsoring us</a>!</p>
```

Comment:
Looks great! Last thing I'd say is that it would probably be better to just put this class on the `<p>` but otherwise this looks great, thanks!